tornjak-frontend/src/components/tornjak-helper.js

```diff
@@ -98,19 +89,19 @@ class TornjakHelper extends Component {
     // get status
     var status = this.SpiffeHelper.getAgentStatusString(agent);
     // get tornjak metadata
-    var metadata_entry = this.SpiffeHelper.getAgentMetadata(thisSpiffeid, workLoadAttestorInfo);
+    var metadataEntry = this.SpiffeHelper.getAgentMetadata(thisSpiffeid, workLoadAttestorInfo);
     var plugin = "None"
     var cluster = "None"
-    if (typeof metadata_entry["plugin"] !== 'undefined' && metadata_entry["plugin"].length !== 0) {
-      plugin = metadata_entry["plugin"]
+    if (typeof metadataEntry["plugin"] !== 'undefined' && metadataEntry["plugin"].length !== 0) {
```

Comment:
alternative could be:
```
if (metadataEntry["plugin"]?.length) {
```

Reply:
File not of concern on this PR. Will be addressed in the next consecutive PR.